_config/site.yml

```diff
@@ -211,17 +211,10 @@ profiles:
       <<: *deploy
       path: /stg_htdocs/it-rhd-stg-main
 
-  # Octopus integration configuration
```

Comment:
Yeah, i removed it as it's redundant and would have required code changes as part of this refactor. It's in a separate commit.